imports/plugins/included/default-theme/client/styles/cards.less

```diff
@@ -83,7 +83,7 @@
 .rui.card .content {
   .flex(1 1 auto);
   padding: 10px;
-  cursor: pointer;
+  cursor: default;
```

Comment:
Are you suggesting that you don't think the card should be clickable at all? Just the little > section?